examples/amp-wpm-player.amp.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-wpm-player example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-wpm-player {
+      color: red;
+    }
+  </style>
+  <script async custom-element="amp-wpm-player" src="https://cdn.ampproject.org/v0/amp-wpm-player-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-iframe"
+    src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js">
+  </script>
+</head>
+<body>
+  Minimal component usage:
+  <br/>
+  <amp-wpm-player
+    layout="responsive"
+    width="1920"
+    height="1080"
+    url="https://wp.tv/?mid=2013067"
```

Comment:
Would you mind adding `url` and `poster` to [`requiredAttrs`](https://github.com/ampproject/amphtml/blob/master/build-system/app-video-testbench.js#L30) and `amp-wpm-player` to [`availableExtensions`](https://github.com/ampproject/amphtml/blob/master/build-system/app-video-testbench.js#L98)?